zlp-scheduler/features/step_definitions/page_steps.rb

```diff
@@ -9,6 +9,22 @@
     expect(current_path).to eq "/admin/view_term_admin"
   elsif page_name == 'open semester'
     expect(current_path).to eq "/admin/open_semester"
+  elsif page_name == 'view cohort semester'
+    expect(current_path).to eq "/admin/view_cohort_semester"
+  else
+    fail("not valid page name")
+  end
+end
+
+Then(/^I should see the page for (.+)$/) do |page_name|
```

Comment:
This does work, but it'd be a little more reusable if you did `/^I should see the (.+) page for (.+)$/` and use the first value as the page and the second value as the name of the cohort to find. It's up to you if you want to make this change.